.eslintrc.js

```diff
@@ -267,6 +267,7 @@ module.exports = {
         'extensions/**/test-e2e/*.js',
         'ads/**/test/**/*.js',
         'testing/**/*.js',
+        'examples/**/*.js',
```

Comment:
Remove this, since it has its own set of overrides below.

Reply:
[See comment below.](#discussion_r568033762)

Reply:
This makes sense to me. PTAL.

(One unfortunate disadvantage is that every rule needs to be disabled at the set at the bottom for `examples/**/*`. It would be great if we could configure it to disable all but `prettier`, can't figure out if that's possible.)